monai/handlers/regression_metrics.py

```diff
@@ -29,7 +29,8 @@ def __init__(
         """
 
         Args:
-            output_transform: transform the ignite.engine.state.output into [y_pred, y] pair.
+            output_transform: callable to construct the `[y_pred, y]` pair from `ignite.engine.state.output`, where
```

Comment:
Same suggestion as written in `ignite_metric.py`.

Reply:
updated in the latest commit.
Thanks.